.github/workflows/actions.yml

```diff
@@ -358,7 +361,6 @@ jobs:
             -DCMAKE_BUILD_TYPE=RelWithDebInfo \
             -DCMAKE_INSTALL_PREFIX=pfx \
             -DCMAKE_OSX_ARCHITECTURES='x86_64' \
-            -DCMAKE_PREFIX_PATH=`brew --prefix`/opt/qt \
```

Comment:
We had it in macos-11 and macos-14, why removing it for macos-12?

Reply:
Removed it elsewhere it is not needed.
Also disabled GTK and Qt clients explicitly in this particular workflow.
Thanks!